js/adapt-drop.js

```diff
@@ -1,93 +1,15 @@
-define([ 'core/js/adapt' ], function(Adapt) {
+import DropView from './DropView';
+import Adapt from 'core/js/adapt';
 
-  class DropView extends Backbone.View {
+Adapt.once('app:dataReady', () => {
```

Comment:
```js
class AdaptDrop extends Backbone.Controller {

  initialize() {
    this.listenTo(Adapt, 'app:dataReady', this.onDataReady);
  }
  
  onDataReady() {
    const views = [ 'menu', 'menuItem', 'page', 'article', 'block', 'component' ];
    const eventList = views.map(view => `${view}View:postRender`).join(' ')
    this.listenTo(Adapt, eventList, this.onEvent);
  }
  
  onEvent(model, $el) {
    const config = model.get('_drop');
    if (!config?._isEnabled) return;
    new DropView({ model, $content: $el.children().first() });
  }

}

export default new AdaptDrop();
```

}

Reply:
@oliverfoster implemented with one difference, curly braces needed in onEvent function to access model & $el
`onEvent({model, $el})`